dask_cuda/benchmarks/utils.py

```diff
@@ -222,14 +229,36 @@ def get_scheduler_workers(dask_scheduler=None):
     return dask_scheduler.workers
 
 
-def setup_memory_pool(pool_size=None, disable_pool=False):
+def setup_memory_pool(
+    dask_worker=None, pool_size=None, disable_pool=False, log_directory=None,
+):
     import cupy
 
     import rmm
 
+    logging = log_directory is not None
+
     if not disable_pool:
         rmm.reinitialize(
-            pool_allocator=True, devices=0, initial_pool_size=pool_size,
+            pool_allocator=True,
+            devices=0,
+            initial_pool_size=pool_size,
+            logging=logging,
+            log_file_name=os.path.join(
+                log_directory,
+                "rmm_log_%s.txt"
+                % (
+                    (
+                        dask_worker.name.split("/")[-1]
+                        if isinstance(dask_worker.name, str)
+                        else dask_worker.name
+                    )
+                    if hasattr(dask_worker, "name")
+                    else "scheduler"
+                ),
+            )
+            if logging
+            else None,
```

Comment:
Could we move this into a function in https://github.com/rapidsai/dask-cuda/blob/branch-0.19/dask_cuda/utils.py ? I see the same is used again below, so better to centralize that.

Reply:
Sure!